src/Microsoft.IdentityModel.Clients.ActiveDirectory/TokenCache.cs

```diff
@@ -207,6 +207,33 @@ public void Deserialize(byte[] adalState)
             }
         }
 
+        /// <summary>
+        /// 
```

Comment:
do you need information here?

Reply:
will add documentation when we finalize the API signatures.  this is in there for Henrik to get access to the serialization methods so we can finalize our compat/migration stories

Reply:
JM was going to do API documentation.

---
In reply to: [255733681](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1500#discussion_r255733681) [](ancestors = 255733681)